addons/api/addon/generated/models/session-recording.js

```diff
@@ -112,12 +112,14 @@ export default class GeneratedSessionRecordingModel extends BaseModel {
 
   @attr('date', {
     description: 'The time until a session recording is required to be stored.',
+    readOnly: true,
```

Comment:
None blocking comment, clarification: I understand setting these attrs to `readOnly` within session-recording is just for show purposes, is that correct? otherwise, we rely on the [policy model](https://github.com/hashicorp/boundary-ui/blob/main/addons/api/addon/generated/models/policy.js#L48-L60).

Reply:
both are different fields, these are only for session recording API